aqua-examples/near-integration/README.md

```diff
@@ -63,7 +71,38 @@ async function main() {
     // ...
 ```
 
-For the complete implementation details, see `src/index.ts`. Before we test our code, please note that in this implementation the wallet credentials are presumed to be in the `~/.near-credentials` directory of the machine/system that runs the Fluence Signing Node. For *testnet* wallets, see https://wallet.testnet.near.org/ and https://docs.near.org/docs/develop/basics/create-account, to get started.
```

Comment:
Why do we need to update the command? It's still relevant since we still compiling the aqua code. The only difference is we specify the `fluence` call instead of `aqua` call in `package.json`. On the other hand the experience of a js/ts developer is still the same with `npm`

Reply:
the fewer tools people have to touch, the better and fluence cli is supposed to be the only cli to rule them all. might as well drop all aqua and marine cli references when fluence cli is in play.

Reply:
in the section, there is also some commented out code, probably from me, `// return Promise.resolve(tx_receipt);` which needs to go.

Reply:
ok, removed it